notebooks/output.sif

```diff
@@ -0,0 +1,413 @@
+EFNA5_HUMAN	N	EPHA7_HUMAN
```

Comment:
Is this the same example input file as `visualization_scripts/output.sif`?  If yes, we can remove this copy from the pull request.

It looks like there are two copies of the output.sif, output.tsv, and style.xml files and that we only need one copy to be merged here.